src/HttpRepository/AirtableHttpRepository.php

```diff
@@ -113,15 +103,21 @@ public function findTheLatest(string $table, $field, ?string $dataClass = null):
     public function addOneRecord(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
     {
         $url = sprintf(
-            '%s/%s',
-            $this->id,
+            '/%s',
             $table
         );
-        $response = $this->request('POST', $url, $fields);
+
+        $response = $this->httpClient->request(
+            'POST',
+            $url,
+            ['json' => [
+                'fields' => $fields, ],
+            ]
+        );
 
         $recordData = $response->toArray();
 
-        if (null == $recordData) {
```

Comment:
`toArray();` cannot be null, it returns an array, don't hesitate to open methods inside the vendor and read code and output :+1: